--- sidebar.html	2023-07-24 17:40:35.000000000 +0200
+++ sidebar.html.microdep	2023-08-03 15:13:10.924413795 +0200
@@ -16,6 +16,7 @@
                 <h3 class="sidebar__heading"><i class="fa fa-bookmark"></i> &nbsp; Other services</h3>
                 <ul class="sidebar-list">
                     <li class="sidebar-list__item"><a href="http://stats.perfsonar.net/" target="_blank">Global node directory <i class="fa fa-external-link"></i></a></li>
+                    <li class="sidebar-list__item"><a href="/microdep/" target="_blank">Microdep analytics <i class="fa fa-internal-link"></i></a></li>
                     
                      [% IF authenticated %]
                          <li class="sidebar-list__item"><a href="/toolkit/admin/plot.cgi">Plot-Schedule Graph</a></li>
